gulpfile.js

```diff
@@ -214,6 +214,11 @@ const mappings = {
     'modeler': 'CompositeSwagger',
     'clientName': 'OperationalInsightsManagementClient'
   },
+  'postgresql': {
+    'packageName': 'azure-arm-postgresql',
+    'dir': 'postgresqlManagement/lib',
+    'source': 'arm-rdbms/2017-04-30-preview/swagger/Rdms.PostgreSQL.json'
```

Comment:
Note: There is one other parameter that could be in play here. You'd notice in some other places in this file (e.g. line 242), there is a parameter called 'FT'. Its called flattening threshold and its passed to autorest for code gen. What it controls is the signature of the generated APIs. notice that most APIs take an `options` object which is a property bag. But if there was only one property inside of it, say for e.g: `serverName`, then the API would be much cleaner if we surfaced `serverName` to the top rather than having an `options` object with 1 property inside it. 

There is no rule of thumb that dictates what FT one should be using, its more of an art, where we look at the generated API and see if it makes sense to flatten some objects. As a consumer of this API, you may discover this need when you're programming against it.

For what its worth, I looked at the shape of the generated API and it looks good as is to me. So we don't yet have to think about flattening and can revisit it later if required.